app/components/wds-global-navigation.js

```diff
@@ -13,6 +13,26 @@ export default Component.extend({
 
 	searchIsAlwaysVisible: empty('model.fandom_overview'),
 
+	track(/*label*/) {
+		// Needs to be passed from client
```

Comment:
I think it shouldn't be here, it should only be passed to component. Same as `searchSuggestionChosen` and `goToSearchReasults`